makePoplogCommander.sh

```diff
@@ -29,51 +54,81 @@ cat << \****
 #endif
 
 //  Bit-flags.
-#define RUN_INIT_P          1
-#define INHERIT_ENV         2
-//  Bit-flag sets.
-#define PREFER_SECURITY     0
+#define RUN_INIT_P          0x1
+#define INHERIT_ENV         0x2
+#define VARIANT_X           0x4
+#define VARIANT_MOTIF       0x8
+//  Bit-flag sets for run vs dev.
+#define RUN_FLAGS           0x3
+#define PREFER_SECURITY     0x0
 #define PREFER_FLEXIBILITY  (RUN_INIT_P|INHERIT_ENV)
+//  Bit-flag sets for variants.
+#define VARIANT_FLAGS       0xC
```

Comment:
Why? Is this meant to be a bit mask that covers both `VARIANT_X` and `VARIANT_MOTIF`? If so, it'd be a lot easier to read as `VARIANT_X | VARIANT_MOTIF`

Reply:
I have adopted this suggestion after a little thought. The question is why both these have the same definition but mean something utterly different:
```
#define VARIANT_FLAGS       (VARIANT_X | VARIANT_MOTIF)
#define VARIANT_XM          (VARIANT_X | VARIANT_MOTIF)
```
The use of 0xC distinguished VARIANT_FLAGS as part of the implementation rather than the abstraction. But, on reflection, one can argue there are two layers of implementation. Since it is easier to read and it seems the implementation/abstraction argument is ambiguous, I have adopted it.